app/Models/Model.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use App\CustomClasses\Builder;
+use Illuminate\Database\Eloquent\Model as EloquentModel;
+
+class Model extends EloquentModel
+{
+  /**
+   * Create a new Eloquent query builder for the model.
+   *
+   * @param  \Illuminate\Database\Query\Builder  $query
+   * @return \Illuminate\Database\Eloquent\Builder|static
+   */
+  public function newEloquentBuilder($query)
+  {
```

Comment:
I think a command should be created to create models that extend from this base model

Reply:
I modified the base file laravel uses to create new model files. So Laravel is taking this custom model for creating new ones